docs/translations/api-docs/use-select/use-select.json

```
@@ -1,6 +1,21 @@
 {
   "hookDescription": "",
-  "parametersDescriptions": {},
+  "parametersDescriptions": {
+    "buttonRef": "The ref of the trigger button element.",
+    "defaultOpen": "If <code>true</code>, the select will be initially open.",
+    "defaultValue": "The default selected value. Use when the component is not controlled.",
+    "disabled": "If <code>true</code>, the select is disabled.",
+    "listboxId": "<code>id</code> attribute of the listbox element.",
+    "listboxRef": "The ref of the listbox element.",
+    "multiple": "If <code>true</code>, selecting multiple values is allowed.\nThis affects the type of the <code>value</code>, <code>defaultValue</code>, and <code>onChange</code> props.",
```

To avoid passive voice.

```suggestion
    "multiple": "If <code>true</code>, the end user can select multiple values.\nThis affects the type of the <code>value</code>, <code>defaultValue</code>, and <code>onChange</code> props.",
```